docs/resources/aws_ec2_image.md

```diff
@@ -0,0 +1,90 @@
+---
+title: About the aws_ec2_image Resource
```

Comment:
Can we please rename this to correspond to the terraform resource? 
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ami

Reply:
Should the attributes then be updated from `image_id` to `ami_id`? Seems like the mix between the two might be a bit confusing.

Reply:
the api call uses image id which is why I continued to use it. 'aws ec2 describe-images --image-id ami-0a13d44dccf1f5cf6'

Reply:
Latest commit renamed to aws_ami